### ISSUE TITLE
Chapter 01 and 03  Karate Club Data accuracy(training and test) for 4 classes

### ISSUE THREAD

**Issue by dgcovell** [PacktPublishing/Hands-On-Graph-Neural-Networks-Using-Python#11]
Working through the excellent chapters to expand my python expertise. Chapters 01 and 03 have sections that process the
Karate Club data using 2 different methods. I would like to compare the training and prediction accuracies for both procedures.
The chapter 01 GNN analysis is based on 2 classes (Mr. Hi vs Officer) while the wordvec2 procedure in chapter 03 has 4 classes.
While the test accuracies for the 2 class case are ~98%, however when I apply the 4 class data, the training accuracy drops to 59%.
I am not sure whether my approach is correct. 

In short, could an analysis of both procedures be provided for the Karate Club data with 4 classes? This would address differences 
between GNN and wordvec2.

Thanks,
DGC

**Comment by mlabonne** [comment]
Thanks @dgcovell! Do you have code you can share with the 4-class example?
